openupgrade_scripts/scripts/project/16.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,99 @@
+obsolete model project.delete.wizard [transient]
+# NOTHING TO DO: wizard removed
+
+---Fields in module 'project'---
+project      / account.analytic.tag     / task_ids (many2many)          : DEL relation: project.task
+project      / project.task             / analytic_tag_ids (many2many)  : DEL relation: account.analytic.tag
+# NOTHING TO DO: lost feature
+
+project      / project.milestone        / task_ids (one2many)           : NEW relation: project.task
+project      / project.project          / allow_milestones (boolean)    : NEW hasdefault: default
+project      / project.project          / task_properties_definition (properties_definition): NEW
+project      / project.task             / task_properties (properties)  : NEW hasdefault: compute
+project      / res.company              / analytic_plan_id (many2one)   : NEW relation: account.analytic.plan, hasdefault: compute
+# NOTHING TO DO: new features
+
+project      / project.tags             / project_ids (many2many)       : NEW relation: project.project
+project      / project.tags             / task_ids (many2many)          : NEW relation: project.task
+# NOTHING TO DO: reverse relation that already exists on project.project and project.task
+
+pad_project  / project.project          / description_pad (char)        : DEL
+pad_project  / project.project          / use_pads (boolean)            : DEL
+pad_project  / project.task             / description_pad (char)        : DEL
+# TODO: Module pad_project has been removed
+
+project      / project.project          / last_update_status (selection): not a function anymore
+project      / project.project          / last_update_status (selection): now required
+project      / project.project          / last_update_status (selection): selection_keys is now '['at_risk', 'off_track', 'on_hold', 'on_track', 'to_define']' ('['at_risk', 'off_track', 'on_hold', 'on_track']')
+# DONE: pre-migration: Set value of last_update_status to 'to_define' where field is empty
+
+project      / project.task.type        / is_closed (boolean)           : DEL
+# DONE: pre-migration: Field removed, field 'fold' should be used instead. Moving values from 'is_closed' to 'fold'.
+
+project      / project.task             / is_closed (boolean)           : is now stored
+# DONE: pre-migration: Add new column & set value for it
+
+project      / project.task             / ancestor_id (many2one)        : NEW relation: project.task, isfunction: function, stored
+project      / project.task             / is_analytic_account_id_changed (boolean): NEW isfunction: function, stored
+# DONE: pre-migration: pre-compute value for new computed fields
+
+project      / project.task             / is_blocked (boolean)          : NEW isfunction: function, stored
+project      / project.task             / milestone_id (many2one)       : NEW relation: project.milestone, hasdefault: compute
+# TODO (speed improvement): pre-migration: pre-compute value for new computed fields
```

Comment:
Is this going to be done?

Reply:
No, I'm not going to do this. I'm focus on getting something that works.